skbio/io/format/tests/data/stockholm_multiple_references

```diff
@@ -0,0 +1,20 @@
+# STOCKHOLM 1.0
```

Comment:
Can you expand this example to include a couple of run-on tags, with and without padding, to demonstrate that it works with multiple references? I'd also make the tags have different values across references, so that you'd catch a bug if the code added data to the wrong reference.